packages/vue/.eslintignore

```diff
@@ -0,0 +1,2 @@
+vue-component-lib/
+src/components/Overlays.ts
```

Comment:
Why was `Overlays.ts` ignored? (Ditto for its inclusion in `.prettierignore`)

Reply:
Overlays.ts is autogenerated